homeassistant/components/homekit/accessories.py

```diff
@@ -147,6 +147,11 @@ def get_accessory(  # noqa: C901
             and features & CoverEntityFeature.SET_POSITION
         ):
             a_type = "Window"
+        elif (
+            device_class == CoverDeviceClass.DOOR
+            and features & CoverEntityFeature.SET_POSITION
```

Comment:
This is contrary to your implementation. The implementation only supports open / close (it even says so in the comment), but here you are checking if the entity only has set_position which is absolute positioning of the cover.

Reply:
Whoops, should be fixed in https://github.com/home-assistant/core/pull/80741/commits/ed6e7227d7927fb3b5d37d8637df3cfe037c3dc3